rust/bridge/shared/Cargo.toml

```diff
@@ -15,7 +15,7 @@ libsignal-protocol = { path = "../../protocol" }
 signal-crypto = { path = "../../crypto" }
 device-transfer = { path = "../../device-transfer" }
 libsignal-bridge-macros = { path = "macros" }
-aes-gcm-siv = { version = "0.10.1", features = ["armv8"] }
+aes-gcm-siv = { version = "0.10.1" }
```

Comment:
This can go back to the short form.

Reply:
> > I've added rust-stable and rust-nightly as separate jobs. Stable doesn't run clippy or `./bin/verify_crate_versions.py`, and doesn't use `-Z unstable-options`. This way, the tests should run in parallel, not sure whether that's what you would have done.
> 
> It's not what I was thinking, since there are organization-level limits on the number of jobs you can run at once, but after checking internally I think it's reasonable. Thanks!

If you would rather have me add an additional install block and run it in the same job, I'll gladly change it!

> > Is it also an idea to drop the `rust-toolchain` file now? Then I probably have to swap `+stable` for `+nightly` somehow.
> 
> The `rust-toolchain` file is effectively a lockfile for us just as much as Cargo.lock: it makes the guarantee that _this_ commit will build with _this_ version of nightly. That's important for historical tags and especially for the Dockerized Android build, which is intended to be reproducible by anyone who wants to verify that what's in the Android app is what we said it is. And it won't get in the way of anyone using libsignal-client crates as dependencies.

Ack, sounds fair. Only "problem" with it in my eyes, is that this triggers a download of an (often out-of-date) nightly chain for developers that are external to Signal (like me), who often just want to use stable.